spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BinderAwareChannelResolver.java

```diff
@@ -37,22 +37,23 @@
  */
 public class BinderAwareChannelResolver extends BeanFactoryMessageChannelDestinationResolver {
 
-	private final ChannelBindingService channelBindingService;
+	private final BindingService bindingService;
 
-	private final BindableChannelFactory bindableChannelFactory;
+	private final AbstractBindingTargetFactory<? extends MessageChannel> boundElementFactory;
```

Comment:
"boundElementFactory" should be replaced with "bindingTargetFactory" throughout this class